src/interpreter/mod.rs

```diff
@@ -118,8 +118,8 @@ enum CachedMir<'mir, 'tcx: 'mir> {
 
 /// Represents the action to be taken in the main loop as a result of executing a terminator.
 enum TerminatorTarget {
-    /// Make a local jump to the given block.
-    Block(mir::BasicBlock),
+    /// Make a local jump to the next block
+    Block,
```

Comment:
Well... it caused bugs in the Stepper because I had to very carefully figure out where to get the next block from. Now that I use `Frame::next_block` this might be way less relevant...

I think we might be able to get rid of `TerminatorTarget` completely